packages/paste-website/src/components/site-wrapper/site-header/HamburgerToggle.tsx

```diff
@@ -1,53 +1,60 @@
 import * as React from 'react';
-import {Box, BoxProps} from '@twilio-paste/box';
-import {BackgroundColor} from '@twilio-paste/style-props';
+import {styled, themeGet} from '@twilio-paste/styling-library';
+import {Box} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+import type {TextColorOptions} from '@twilio-paste/style-props';
 
 interface HamburgerToggleProps {
   toggled: boolean;
-  color: BackgroundColor;
+  color: TextColorOptions;
 }
 
-const Pattie: React.FC<BoxProps> = (props) => (
-  <Box
-    as="span"
-    position="absolute"
-    width="16px"
-    height="3px"
-    borderRadius="borderRadius30"
-    transition="all .25s ease-in-out"
-    transformOrigin="left center"
-    {...props}
-  />
-);
+interface StyledPattieProps extends Pick<BoxProps, 'top' | 'left' | 'transform' | 'opacity'> {
+  color: TextColorOptions;
+}
+
+const StyledPattie = styled.span<StyledPattieProps>`
```

Comment:
suggestion: we've been moving away from themeGet to the styled+css combo for performance and ease of development. Typing out `${themeGet()}` all over gets messy heh

```js
const StyledPattie = styled.span<StyledPattieProps>(({color, ...props}) =>
  css({
    position: 'absolute',
    width: 'space.space50',
    height: '3px',
    borderRadius: 'radii.borderRadius30',
    transition: 'transform 0.25s ease-in-out, opacity 0.25s ease-in-out',
    transformOrigin: 'left center',
    backgroundColor: `textColors.${color}`,
    ...props
  })
);
```

Reply:
I initially tried doing it this way, but I think the CSS function doesn't let you use text colors for background color. I just tried it again and It doesn't set the background color at all:
<img width="406" alt="Screen Shot 2022-04-06 at 11 11 19 AM" src="https://user-images.githubusercontent.com/8482334/162040724-14f76900-3e6c-4a6f-a07f-188f59cc1593.png">

Reply:
Yeah, it won't. The CSS function is keyed to the theme and style props. So you can only use backgrounds for background and colors for colors. Mainly, so you don't do something that isn't intended by the design language. 

Here it's fine, as you are building an icon from backgrounds. It's a funky edge case

Reply:
I see, I hadn't noticed that.